PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: hodasemi <michaelh.95 at t-online dot de>
+_pkgbase=XRLinuxDriver
+pkgname="${_pkgbase}"
```

Comment:
Let's give this a name that's specific to BreezyGNOME for now (e.g. `XRLinuxDriver-BreezyGNOME`). In the future I intend to have the plugins installed as separate dependencies, like how GStreamer has a base install and then separate plugin installs, so I would want `XRLinuxDriver` to be the base installation. In the interim, some way to distinguish this package from that base package I think would be nice.